src/redteam/plugins/index.ts

```diff
@@ -71,6 +71,26 @@ type PluginClass<T extends PluginConfig> = new (
   config: T,
 ) => RedteamPluginBase;
 
+/**
+ * Computes modifiers from config (same logic as appendModifiers in base.ts).
+ * Used to ensure modifiers are available for strategies when using remote generation.
+ */
+function computeModifiersFromConfig(config: PluginConfig | undefined): Record<string, string> {
+  const modifiers: Record<string, string> = {
+    ...(config?.modifiers as Record<string, string> | undefined),
+  };
+  if (config?.language && typeof config.language === 'string') {
+    modifiers.language = config.language;
+  }
+  if (config?.inputs && Object.keys(config.inputs).length > 0) {
+    const schema = Object.entries(config.inputs as Record<string, string>)
+      .map(([k, description]) => `"${k}": "<${description}>"`)
```

Comment:
Same comment re: arrow brackets. DRY this into a function w/ an explainer docstring + unit test cov?